compiler/lib/driver.ml

```diff
@@ -397,6 +398,35 @@ let pack ~global { Linker.runtime_code = js; always_required_codes } =
   in
   let runtime_js = wrap_in_iifa ~can_use_strict:true js in
   let js = List.flatten always_required_js @ runtime_js in
+  let js =
+    match global, standalone with
+    | (`Function | `Bind_to _ | `Custom _), _ -> js
+    | `globalThis, false -> js
+    | `globalThis, true ->
+        let s =
+          {|
+(function (Object) {
+  typeof globalThis !== 'object' && (
+    this ?
+      get() :
+      (Object.defineProperty(Object.prototype, '_T_', {
+        configurable: true,
+        get: get
+      }), _T_)
+  );
+  function get() {
+    var global = this || self;
+    global.globalThis = global;
```

Comment:
Ah, I get this answers my question above :) no need to release major version as backwards compat is guaranteed.

Reply:
indeed